CHANGES.rst

```diff
@@ -68,6 +68,10 @@ Release history
   values. (`#1609`_)
 - The ``nl`` and ``nl_nodirect`` test arguments have been renamed to ``AnyNeuronType``
   and ``NonDirectNeuronType``. (`#1609`_)
+- Weight solvers (i.e. those with ``weights=True``) now work on connections from
+  ``Node`` to ``Ensemble``. They also work on connections from ``Ensemble`` to ``Node``,
+  but issue a warning, since here they are identical those with ``weights=False``.
```

Comment:
should be "identical _to_ those with"

Reply:
And I was so happy with my line being under 88 characters.

Reply:
Haha... for changelogs I still like to keep it something more like 80 characters, though not that strictly. :shrug: